File: app/src/main/java/com/fintech/superadmin/data/browseplan/Info.java
package com.fintech.superadmin.data.browseplan;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Info {
    @SerializedName("COMBO")
    public List<Plan> cOMBO;
    @SerializedName("FULLTT")
    public List<Plan> fULLTT;
    @SerializedName("2G")
    public List<Plan> g;
    @SerializedName("3G/4G")
    public List<Plan> g4G;
    @SerializedName("RATE CUTTER")
    public List<Plan> rATECUTTER;
    @SerializedName("Romaing")
    public List<Plan> romaing;
    @SerializedName("SMS")
    public List<Plan> sMS;
    @SerializedName("TOPUP")
    public List<Plan> tOPUP;

    @SerializedName("LOCAL")
    public List<Plan> lOCAL;

    @SerializedName("OTHER")
    public List<Plan> oTher;
}

File: app/src/main/java/com/fintech/payware/data/deer_response/Info.kt
package com.fintech.payware.data.deer_response


import com.google.gson.annotations.SerializedName

data class Info(
    @SerializedName("COMBO")
    val cOMBO: List<Plan>,
    @SerializedName("FULLTT")
    val fULLTT: List<Plan>,
    @SerializedName("2G")
    val g: List<Plan>,
    @SerializedName("3G/4G")
    val g4G: List<Plan>,
    @SerializedName("RATE CUTTER")
    val rATECUTTER: List<Plan>,
    @SerializedName("Romaing")
    val romaing: List<Plan>,
    @SerializedName("SMS")
    val sMS: List<Plan>,
    @SerializedName("TOPUP")
    val tOPUP: List<Plan>
)